### DISCUSSION TITLE
Blog:Adding the daily DevCard to your GitHub profile needs update

### DISCUSSION THREAD

**Discussion by fatihcaliss** [dailydotdev/daily#1077] [General]
I read [Adding the daily DevCard to your GitHub profile needs update](https://daily.dev/blog/adding-the-daily-devcard-to-your-github-profile?utm_source=webapp&utm_medium=devcard&utm_campaign=devcardguide&utm_id=inapp) article. I think it would be good to refresh this content because of new version [dailydotdev/action-devcard@3.0.0](https://github.com/dailydotdev/action-devcard)  is released and it has some changes as for  example: "devcard_id" section here should be changed to "user_id".

**Comment by idoshamun** [comment]
Hi @fatihcaliss,

Thanks for the report. We'll fix it asap

**Reply by idoshamun** [reply]
@fatihcaliss the blog is now up to date. Thanks a lot!

**Comment by keshavashiya** [comment]
@idoshamun How can we use Horizontal card in README file?

**Reply by idoshamun** [reply]
Select the horizontal mode and the embed code will update to reflect this change :)

**Reply by keshavashiya** [reply]
@idoshamun I meant with Github Action as we don't use params in GH Action how we've for embedded url.

**Reply by fatihcaliss** [reply]
@keshavashiya,  I think you can do it with new release version 3.1.0 [https://github.com/dailydotdev/action-devcard ](https://github.com/dailydotdev/action-devcard?tab=readme-ov-file#optional):

`type`: Configure orientation for devcard.
 - `default`: Vertical (Default)
 - `wide`: Horizontal`
